src/main/java/com/meilisearch/sdk/SettingsHandler.java

```diff
@@ -90,10 +86,12 @@ public String[] getRankingRuleSettings(String uid) throws Exception {
      * @throws Exception if an error occurs
      */
     public Task updateRankingRuleSettings(String uid, String[] rankingRules) throws Exception {
-        String rankingRulesAsJson = gson.toJson(rankingRules);
-        return this.gson.fromJson(
+        return meilisearchHttpRequest.jsonHandler.decode(
                 meilisearchHttpRequest.post(
-                        "/indexes/" + uid + "/settings/ranking-rules", rankingRulesAsJson),
+                        "/indexes/" + uid + "/settings/ranking-rules",
+                        rankingRules == null
+                                ? meilisearchHttpRequest.jsonHandler.encode(rankingRules)
```

Comment:
If you just encode null, does not work?

Reply:
Yes, it works but I find it dirty no?
Actually I can write:
```
                        rankingRules == null
                                ? meilisearchHttpRequest.jsonHandler.encode(null)
                                : rankingRules),
```
And even:
```
                        rankingRules == null ? "null" : rankingRules),
```
What do you think is the best?

Reply:
> Yes, it works but I find it dirty no?

I don't think so if the idea is to send an empty array of strings the result should be `[]` 🤔

Reply:
The idea is that when you send "null", you can reset the ranking rules setting so that it returns to its default state:
```
[
	"words",
	"typo",
	"proximity",
	"attribute",
	"sort",
	"exactness"
]
```
If we send an empty array it will become:
```
[]
```